### PR TITLE
Pro 2489 content changes

### PR DESCRIPTION
**Pull request by RebeccaBaker**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/PRO-2489

### Change description ###
Content changes as discussed with Leigh, Max and Nadeem

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Comment by venumadhavancha** [comment]
@sanjayparekh-hmcts please wait merging this PR, I just need to understand the changes and see the testing impact.

File: src/main/resources/templates/markdown/nextSteps.md [hunk 1]
```diff
@@ -26,23 +26,28 @@ Payment summary
...
+Notes for this application
+-------------------------------------------------
+
+{{additionalInfo}}
+
+What you need to do now
 ==================================================
 
-You will now need to send us:
+Print a copy of this page for your records.
```
Comment (sanjayparekh-hmcts) [inline src/main/resources/templates/markdown/nextSteps.md]: we are just awaiting review from Janet regarding additional information on HOW to print this page, will discuss with her tomorrow and update this PR

Comment (sanjayparekh-hmcts) [reply src/main/resources/templates/markdown/nextSteps.md]: Confirmed from Janet, no additional information is required

File: src/test/java/uk/gov/hmcts/probate/service/ConfirmationResponseServiceFeatureTest.java [hunk 2]
```diff
@@ -59,56 +60,130 @@
...
+                .lastname("Smith")
+                .reasonNotApplying(REASON_FOR_NOT_APPLYING_RENUNCIATION)
                 .build();
+        Executor deadExecutor = Executor.builder()
+                .forename("Bob")
+                .lastname("Martin")
+                .reasonNotApplying(REASON_FOR_NOT_APPLYING_DIED_BEFORE)
+                .build();
+        CCDData ccdData = createCCDataBuilder()
+                .executors(Lists.newArrayList(renouncingExecutor, deadExecutor))
```
[the remark this answers is not in the document]
Comment (dhruvpatel20) [reply src/test/java/uk/gov/hmcts/probate/service/ConfirmationResponseServiceFeatureTest.java]: Fair point, I have removed them. Always used to using Guava collection factories in my previous 3 projects. Java 9 has some decent ones coming up.

**Pull request merged**